pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -417,12 +417,12 @@ static Value createInit(OpBuilder &builder, Location loc, Type type,
     }
     case AggregationKind::min: {
       auto value = convertFloatUsingType(
-          llvm::APFloat(std::numeric_limits<float>::max()), floatType);
+          llvm::APFloat(std::numeric_limits<double>::infinity()), floatType);
```

Comment:
What if we used `llvm::APFloat::getInf()`? I'm not exactly sure what to pass to the `Sem` argument, but it seems like generating an `APFloat` directly might work for all data types better than this. It's not clear to me what happens when `floatType` is 16, 32, 64, or even larger bitwidths.

Reply:
I switched to `APFloat` with `APFloat::IEEEdouble()` semantics; does that seem better than the numeric_limits approach?

(Looking in `llvm/unittests/ADT/APFloatTest.cpp` suggests IEEEsingle() and IEEEquad() are also options for semantics, although those don't seem better. Presumably other variants of semantics exist as well.)

Reply:
I suppose we could also add a switch for semantics based on the type we get.

Reply:
Ya, I think the `type` has the target float semantics that we want (no need to 'convert')